test/dynamo/test_higher_order_ops.py

```diff
@@ -138,8 +146,15 @@ def _test_wrap_simple(self, func, args, expected_num_wrap_args, expected_opcount
         self.assertEqual(cnt.op_count, expected_opcount)
 
         self.assertEqual(len(backend.graphs), 1)
-        wrap_node = find_first_node(backend.graphs[0], wrap)
+        graph = backend.graphs[0]
+        wrap_node = find_first_node(graph, wrap)
         self.assertEqual(len(wrap_node.args), expected_num_wrap_args)
+        if expected_graph:
+            self.assertExpectedInline(graph.code.strip(), expected_graph)
+        if expected_body_graph:
+            self.assertExpectedInline(
+                graph.wrap_body_0.code.strip(), expected_body_graph
+            )
```

Comment:
There's an automatic way to update the string passed to expecttest: setting the EXPECTTEST_ACCEPT=1 environment variable will update the string. I'm not sure that this works with the current way you have set this up. If it doesn't, then we should have _test_wrap_simple return a graph (if the some flag is True), and then do the assertExpectedInline inside the test_* method.